### PR TITLE
[WIP] [Conflict] Trading Console

### PR DESCRIPTION
**Pull request by buggy213**
I tried my hardest on the art lol

### CODE REVIEW

File: Assets/Scripts/Models/Prototypes/BasePrototypes.cs [hunk 1]
```diff
@@ -93,7 +93,7 @@ public T Get(int index)
     }
 
     /// <summary>
-    /// Adds the given prototype. If the protptype exists it is overwirten.
+    /// Adds the given prototype. If the prototype exists it is overwritten.
```
Comment (TomMalbran) [inline Assets/Scripts/Models/Prototypes/BasePrototypes.cs]: Good catch. Ty

File: Assets/StreamingAssets/Data/Furniture.xml [hunk 2]
```diff
@@ -540,5 +540,26 @@
     <UnlocalizedDescription>furn_rt_generator_desc</UnlocalizedDescription>
   </Furniture>
 
+  <Furniture objectType ="trading_console">
```
Comment (TomMalbran) [inline Assets/StreamingAssets/Data/Furniture.xml]: Try not to leave a space before the =

File: Assets/StreamingAssets/Data/Furniture.xml [hunk 3]
```diff
@@ -540,5 +540,26 @@
...
+    <PathfindingModifier>0</PathfindingModifier>
+    <PathfindingWeight>2</PathfindingWeight>
+    <Width>1</Width>
+    <Height>1</Height>
+
+    <PowerConnection inputRate ="1" outputRate ="" capacity =""/>
+    
+    <BuildingJob jobTime ="0"/>
+    
+    <ContextMenuAction FunctionName="TradingConsole_PassivelyTrade" Text="Passively trade" RequiereCharacterSelected="false" />
```
Comment (TomMalbran) [inline Assets/StreamingAssets/Data/Furniture.xml]: Are you looking to implement these functions? if you do you can add them to Furniture.lua

Comment (buggy213) [inline Assets/StreamingAssets/Data/Furniture.xml]: yeah i will eventually, i'm still trying to figure out the furniture system

Comment (sboigelot) [inline Assets/StreamingAssets/Data/Furniture.xml]: To implement this you will probably have to add logic in the TradeController (in the EvaluateTraderVisit method).

File: Assets/StreamingAssets/Images/Furniture/XMLFile1.xml [hunk 4]
```diff
@@ -0,0 +1 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (Tranberry) [inline Assets/StreamingAssets/Images/Furniture/XMLFile1.xml]: What does this file do?

**Comment by Tranberry** [comment]
On the sprite, is it meant to be free standing screens on the floor? I'm just curious of how you want it to look.

**Comment by buggy213** [comment]
no, it's just that i can't art. i'll make it look better later

File: Assets/StreamingAssets/Images/Furniture/XMLFile1.xml (hunk 4 above)
Comment (buggy213) [inline Assets/StreamingAssets/Images/Furniture/XMLFile1.xml]: i don't know how that got there

**Comment by sboigelot** [comment]
This branch has conflicts that must be resolved

**Pull request closed without merging**